File: B/Java/RAM.java
public class RAM
{
	// private attributes
	private int capacity; 
	private int price; 

	// constructor
	public RAM(int capacity, int price)
	{
		this.capacity = capacity;
		this.price = price;
	}
	public RAM()
	{

	}

	// methods set for capacity and price attribute
	public void setCapacity(int capacity)
	{
		this.capacity = capacity;
	}
	public void setPrice(int price)
	{
		this.price = price;
	}

	// methods get for capacity and price attribute
	public int getCapacity()
	{
		return this.capacity;
	}
	public int getPrice()
	{
		return this.price;
	}
}

File: B/Python/RAM.py
class RAM():
	
	capacity = 0 
	price = 0 
	
	def __init__(self, capacity = 0, price = 0):
		self.capacity = capacity
		self.price = price

	# methods set and get for capacity attribute
	def setCapacity(self, capacity):
		self.capacity = capacity

	def getCapacity(self):
		return self.capacity

    # methods set and get for price attribute
	def setPrice(self, price):
		self.price = price

	def getPrice(self):
		return self.price